client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -79,13 +82,15 @@ const ConnectedSingleSiteJetpack = connectOptions(
 			<div>
 				<SidebarNavigation />
 				<CurrentTheme siteId={ siteId } />
-				<ThemeShowcase { ...props } siteId={ siteId }>
+				<ThemeShowcase { ...props }
+					siteId={ siteId }
+					emptyContent={ showWpcomThemesList ? <div /> : null } >
```

Comment:
This doesn't quite fix things. If I test with `"manage/themes/upload": false` in `development.json`, I still see an empty page for no results.

I think we will need some kind of full config check like below. Maybe `showWpcomThemesList` should include the config check?